LICENSE

```diff
@@ -808,12 +957,27 @@ This product bundles sys/queue.h within librdkafka, which under a 3-clause UC Re
  * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
+ *
+ *  @(#)queue.h 8.5 (Berkeley) 8/20/94
+ * $FreeBSD$
  
 This product bundles regexp.c and regexp.h within librdkafka offered under a public domain license as below.
 
-These libraries are in the public domain (or the equivalent where that is not possible). You can do anything you want with them. You have no legal obligation to do anything else, although I appreciate attribution.
```

Comment:
I think we should keep this line (also in msi/LICENSE.txt)

Reply:
Updated!